### ISSUE TITLE
Install request: ANTs

### ISSUE THREAD

**Issue by heatherkellyucl** [UCL-RITS/rcps-buildscripts#9]
Not on new Legion yet. IN00772718

http://stnava.github.io/ANTs/

**Comment by heatherkellyucl** [comment]
The 2.1.0 release has a problem with ProjectSourceVersionVars.cmake being missing: you need to do:
```
cp $temp_dir/ANTs-${VERSION}/CMake/ProjectSourceVersionVars.cmake.in $temp_dir/ANTs-${VERSION}/CMake/ProjectSourceVersionVars.cmake
```

**Comment by heatherkellyucl** [comment]
Installed and module available as `ants/2.1.0`, needs testing.

**Comment by heatherkellyucl** [comment]
No errors reported.
